### ISSUE TITLE
Font error when not specifying `ogp_image`

### ISSUE THREAD

**Issue by ru-fu** [wpilibsuite/sphinxext-opengraph#107]
When using the extension without specifying a value for `ogp_image`, local builds return the following error:
```
findfont: Font family 'Roboto' not found.
findfont: Font family 'Roboto' not found.
findfont: Font family 'Roboto' not found.
findfont: Font family 'Roboto' not found.
...
```

To reproduce, just create a default project with `sphinx-quickstart` and add the extension to the list in `conf.py`.
The error disappears when specifying a value for `ogp_image`.

Roboto is not installed on my system. I haven't tested if the error disappears when installing the font.

**Comment by TheTripleV** [comment]
What OS are you using?
What is your sphinx version?
What is your sphinx theme?

**Comment by ru-fu** [comment]
> What OS are you using? 
Ubuntu 22.04

> What is your sphinx version? 
4.2.0

> What is your sphinx theme?
It happens with Furo, but also with the default (alabaster).

**Comment by xqt** [comment]
We have the same problem with

- Python 3.9
- Sphinx==7.2.3
- furo==2023.8.19

See:
* https://integration.wikimedia.org/ci/job/pywikibot-core-tox-publish/5647/console
* https://phabricator.wikimedia.org/T333800

**Comment by xqt** [comment]
This issue is solved with `sphinxext-opengraph==0.8.2`

**Comment by sciencewhiz** [comment]
Fixed by #110
